### DISCUSSION TITLE
Badge campaigns related API interface optimisation suggestions

### DISCUSSION THREAD

**Discussion by lalala108** [allegro/allegro-api#9143] [Suggestions - rejected]
For the Get a list of badges-API interface, is it possible to add a new activity ID field to group query data from different activities? Because currently our ERP to get too much product data, otherwise it is not possible to quickly get different activities products and product status, so we need to get through the activity ID grouping in order to better maintain the user data.
![idea](https://github.com/allegro/allegro-api/assets/47216074/21b83033-61a7-4fbc-83b2-b0c4b49dd93c)

**Comment by Lukasz-Zurek** [comment]
@lalala108 Thank your for your suggestion. I am converting your issue to discussion so the other users can vote for it and we  will verify it with another department, as for every discussions topic.

**Reply by lalala108** [reply]
Okay, thank you for your attention.

**Comment by Lukasz-Zurek** [comment]
We have updated the suggestion's status to "rejected" as we have no plans to implement the suggested change.
